modules/cij/runner.py

```diff
@@ -588,10 +646,10 @@ def tsuite_setup(trun: TestRun, declr, enum) -> TestSuite:
     suite.alias = declr.get("alias")
     suite.ident = "%s_%d" % (suite.name, enum)
 
-    suite.res_root = os.sep.join([trun.conf["OUTPUT"], suite.ident])
+    suite.res_root = os.path.join(tplan.res_root, suite.ident)
     suite.aux_root = os.sep.join([suite.res_root, "_aux"])
```

Comment:
I think the change to `path.join` is nice - would make sense to do for aux_root as well. It guess it might make sense to just change all of these throughout the codebase in a separate commit

Reply:
Adding this to a list of cleanups to do later on.